.github/workflows/ci.yml

```diff
@@ -98,12 +98,33 @@ jobs:
       - uses: actions/setup-node@v3
         with:
           node-version: 18.13.0
-      - name: 📦 Install Dependencies
+      - name: 🔍 Check Dependencies
         run: yarn install --immutable --immutable-cache --check-cache
 
+  gql-codegen:
```

Comment:
Can you please put this in a separate commit to ensure commits have one very clear goal? - This also helps future devs when viewing code, as they know why changes happened due to the commit message.